portal/models/intervention.py

```diff
@@ -106,6 +106,20 @@ def as_json(self):
 
         return d
 
+    @staticmethod
+    def rct_ids():
+        """returns list of RCT (randomized control trial) intervention ids"""
+        names = current_app.config.get('RCT_INTERVENTIONS')
+        if not names:
+            return None
+        ids = [i.id for i in Intervention.query.filter(
```

Comment:
Would you be able to just return the `Intervention` `Query`?

You could `filter()` against `~rct_interventions`,  then you wouldn't have to look up the intervention twice (looking up the ids, then filtering on them).